ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -84,7 +85,9 @@ const mapStateToProps = (state, ownProps) => {
 
   const requestState = getRequestState(state, permissionsRequestId) || {};
 
-  const accountsWithLabels = getAccountsWithLabels(state);
+  const accountsWithLabels = getAccountsWithLabels(state).filter((account) =>
```

Comment:
We should use the same comment here (you can update the wording of my suggestion, it was just to give an example):

```suggestion
    // We only consider EVM accounts.
    // Connections with non-EVM accounts (Bitcoin only for now) are used implicitly and handled by the Bitcoin Snap itself.
  const accountsWithLabels = getAccountsWithLabels(state).filter((account) =>
```